kolibri/utils/i18n.py

```diff
@@ -27,7 +27,10 @@ def get_installed_app_locale_path(appname):
 
 def _get_language_info():
     file_path = os.path.join(
-        os.path.dirname(kolibri.__file__), "locale", "language_info.json"
```

Comment:
This path resolution will fail in production, so let's just leave this with the duplicated language_info.json files until we've sorted out how to handle that.